lab-padget/server.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
```

Comment:
as someone who needs comments in their code for their own edification, i applaud your gray comments. as a TA, i have to dock you if i see them in your lab submission.